templates/internet-of-things/robotics.html

```diff
@@ -249,7 +249,7 @@ <h3 class="p-stepped-list__title">
           Set up your development environment
         </h3>
         <div class="p-stepped-list__content">
-          <p>Set up your development toolchain with Ubuntu Server via Multipass or on your Ubuntu Desktop. Install Snapcraft to build a bespoke application stack for your target use case. Benefit from the wealth of tools and libraries available on Ubuntu, and take advantage of the strong community support.</p>
+        <p>Set up your development toolchain with Ubuntu Server via <a href="https://multipass.run/>Multipass"</a> or on your Ubuntu Desktop. Install Snapcraft to build a bespoke application stack for your target use case. Benefit from the wealth of tools and libraries available on Ubuntu, and take advantage of the strong community support.</p>
```

Comment:
This link is broken. Needs to be:

```suggestion
        <p>Set up your development toolchain with Ubuntu Server via <a href="https://multipass.run/">Multipass</a> or on your Ubuntu Desktop. Install Snapcraft to build a bespoke application stack for your target use case. Benefit from the wealth of tools and libraries available on Ubuntu, and take advantage of the strong community support.</p>
```

Reply:
Other than that it's perfect :laughing: